test/integration/interpreter/micropython/storage.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script type="module">
+        import { init } from '../utils.js';
+        init('micropython');
+    </script>
+</head>
+<body>
+  <script type="micropython" async>
+    from polyscript import storage
+
+    map = await storage("@polyscript/storage")
```

Comment:
Perhaps rename `map` to `store`..? To my ears it sounds more natural. Feel free to ignore.

Reply:
it's just an example ... you can name it *s* if you prefer ... I don't mind, though it's literally an `instanceof Map`

Reply:
done